hr_timesheet_hours_month/views/project_task_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_project_task_form" model="ir.ui.view">
+        <field name="name">project.task.form</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='planned_hours']" position="after">
+                <field name="planned_hours_year" string="Planned hours to year"/>
```

Comment:
Add a boolean toggle widget, like this: 

https://github.com/solvosci/slv-field-service/blob/c1a049340b346bc658c001192e5769a4af69334d/fieldservice_person_salary/views/fsm_order_salary_views.xml#L38